src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: $transforDuration;
```

Comment:
Not fixed. You should specify each property. Not only transform
<img width="866" alt="image" src="https://user-images.githubusercontent.com/99403198/230716037-fb57aeed-d2ba-4ae6-8e2a-6a581fc11557.png">
<img width="443" alt="image" src="https://user-images.githubusercontent.com/99403198/230716055-99b88670-435f-4b6c-809e-af77245f004c.png">